Automattic/wp-calypso
# Adds Anti-Spam as subset of Jetpack plans

#### Changes proposed in this Pull Request

* Updates plan constants to include Jetpack Anti-Spam.

#### Screenshots

<img width="515" alt="Screen Shot 2020-07-20 at 1 38 11 PM" src="https://user-images.githubusercontent.com/1760168/87968446-711b4200-ca8e-11ea-802d-6cc1e182c3d7.png">
<img width="1059" alt="Screen Shot 2020-07-20 at 1 38 15 PM" src="https://user-images.githubusercontent.com/1760168/87968448-711b4200-ca8e-11ea-8234-7c6b9444c30b.png">

#### Testing instructions



* Set up a Jetpack site with the Personal or higher plan.
* Navigate to Plan -> Plans in Calypso.
* Verify that Anti-Spam is shown as part of the plan.
* Add Anti-Spam product via admin.
* Verify page lists Anti-Spam in conflict with plan.

Fixes 1181531115069428-as-1182817583001078.
